code-challenges.js

```diff
@@ -1,68 +1,168 @@
+//     }
+// }
+
+// console.log(water(212))
+// console.log(water(350))
+// console.log(water(42))
+// console.log(water("test"))
+
+
+
+
+// // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
+
+
+// // Describe your process:
+// 1. Create a variable called padresRunsarray that will combine the values of two arrays then return the length
+// 2. The function uses the .length property
+// 3. The .length property looks at the index then returns the number of values in the array
+
+
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+// // Expected output: 9
+
+// var padresRunsArray = [2, 5, 2, 2, 4, 6, 3, 5, 3,]
+// console.log(padresRunsArray.length)
+
+
+
+
+// // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
+
+// // Describe your process:
+
+// 1. Create a function called curCohort that will reverse the values of the array
+// 2. Convert the string into an array using the split method
+// 3. Use the reverse method to reverse the values
+
+// const currentCohort = "Golf 2022"
 // Expected output: "2202 floG"
 
-// --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
+// var curCohort = ("Golf 2022")
+
+// var curCohort = ("Golf 2022".split(""))
+// console.log(curCohort)
+// console.log(curCohort.reverse(""))
+
+
+
 
-// Describe your process:
+// // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
-const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
-const givenValue1 = 42
-// Expected output: 7
+// // Describe your process:
+// 1. Create function called numbersArray
+// 2. Use the .lastindexof accessor method
 
-const givenValue2 = 10
-// Expected output: 8
+// const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
+// const givenValue1 = 42
+// // Expected output: 7
 
-// --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
+// const givenValue2 = 10
+// // Expected output: 8
 
-// Describe your process:
+// var numbersArray = [13, 34, 42, 5, 5, 10, 27, 42, 10]
 
-const sanDiegoSummerTemperatures = [79, 80, 72, 73, 82, 77, 76]
+// console.log(numbersArray.lastIndexOf(42)) 
+// console.log(numbersArray.lastIndexOf(10))
+// console.log(numbersArray.lastIndexOf(100)) // -1 because the value does not exist
+
+
+
+
+// // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
+
+// // Describe your process:
+// 1. Create a function called numbers
+// 2. Set up two paramaters called numbers1 and numbers2
+// 3. Use the .sort built in method 
```

Comment:
Make sure to expand on this if you are providing extra information to the sort method that handles some kind of logic.